tests/modules/asset_groups/resources/utils.py

```diff
@@ -450,6 +462,40 @@ def patch_asset_group_sighting(
     return response
 
 
+def patch_asset_group_sighting_as_sighting(
+    flask_app_client,
+    user,
+    patch_path,
+    data,
+    expected_status_code=200,
+    expected_resp='',
+):
+    with flask_app_client.login(user, auth_scopes=('asset_group_sightings:write',)):
+        response = flask_app_client.patch(
+            f'{PATH}sighting/as_sighting/{patch_path}',
+            content_type='application/json',
+            data=json.dumps(data),
+        )
+
+    if expected_status_code == 200:
+        # startTime and locationId are only present in the _as_sighting endpoints,
+        # since they are in the config of a standard AGS
+        test_utils.validate_dict_response(
+            response,
+            200,
+            {'guid', 'stage', 'completion', 'assets', 'startTime', 'locationId'},
+        )
+    elif expected_status_code == 400:
+        test_utils.validate_dict_response(
+            response, expected_status_code, {'status', 'message', 'passed_message'}
+        )
+    else:
+        test_utils.validate_dict_response(
+            response, expected_status_code, {'status', 'message'}
+        )
+    return response
+
+
 def read_asset_group_sighting(
     flask_app_client, user, asset_group_sighting_guid, expected_status_code=200
```

Comment:
rename asset_group_sighting_guid as sighting path and you can remove the method below. 
I've done this with a few of the utils as soon as the sub path bit gets more complex than a simple guid

Reply:
@hwindsor, @drewblount and I chatted and we are not sure what this change will look like. I'm going to merge this PR and when you are back we can sync about it and possibly make a small separate PR.

Reply:
I had another look and I agree, this needs work. Created Dex 483 to sort this lot out. These tests have become a bit of a copy& bodge mess. I think they can be cleaned up quite easily though